extensions/wikia/AdEngine/js/video/vastUrlBuilder.js

```diff
@@ -47,40 +47,30 @@ define('ext.wikia.adEngine.video.vastUrlBuilder', [
 		return aspectRatio >= 1 || !isNumeric(aspectRatio) ? adSizes.horizontal : adSizes.vertical;
 	}
 
-	function buildAdUnit(slotParams, useMegaAdUnitBuilder) {
-		var adUnitBuilder = useMegaAdUnitBuilder ? megaAdUnitBuilder : regularAdUnitBuilder;
-		return adUnitBuilder.build(slotParams.pos, slotParams.src);
+	function getAdUnit(options, slotParams) {
+		if (!options.adUnit && adContext.get('opts.megaAdUnitBuilderEnabled')) {
+			// TODO: remove it after merge & release ADEN-5825 mobile-wiki, this is cache related backward compatibility
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)